readme.md

```diff
@@ -135,23 +138,24 @@ $ npm -v
 
 * **Question:** What is `path` and where does it come from?
 
-* **Your Answer:**
+* **Your Answer:** It is the path of the package location and comes from node
```

Comment:
Strictly speaking, `path` is a package that is imported by default from Node. `path` has _methods_ that will tell us interesting information like the file path.